fusor-ember-cli/app/mixins/deployment-controller-mixin.js

```diff
@@ -81,49 +86,69 @@ export default Ember.Mixin.create({
 
   currentStepNumber: null, //set by setupController,
 
-  // steps
-  stepNumberRhev: 2,
+  numberProducts: Ember.computed('isRhev', 'isOpenStack', 'isCloudForms', 'isOpenShift', function() {
```

Comment:
I re-did the logic to calculate the step numbers. It think it will be easier if we need to add another step rather than having `if` statements on every combination.